### ISSUE TITLE
Document that Block.stmts() does not include last_stmt

### ISSUE THREAD

**Issue by evaera** [Kampfkarren/full-moon#214]
[`Block.stmts()`](https://docs.rs/full_moon/0.14.0/full_moon/ast/struct.Block.html#method.stmts) does not include the `LastStmt` which can be gotten with [`Block.last_stmt()`](https://docs.rs/full_moon/0.14.0/full_moon/ast/struct.Block.html#method.last_stmt). This should be documented in `stmts`

**Comment by sno2** [comment]
I have proposed the specified fix in #248 , although I would have rather changed `Block::stmts` to be something like this:

```rust
pub fn stmts(&self) -> impl Iterator<Item = &Stmt> {
  self.stmts.iter().map(|(stmt, _)| stmt).chain(self.last_stmt.iter().map(|(last_stmt, _)| last_stmt.into_regular_stmt()))
}
```

so that it would include the last statement anyways. However, this would be breaking so the notice would probably be sufficient enough.

**Comment by Kampfkarren** [comment]
I think I considered it at some point but it makes dev UX worse for reasons I don't quite remember. I would rather just a notice.
